packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.ts

```diff
@@ -27,7 +32,8 @@ export const useArbQueryParams = () => {
   return useQueryParams({
     amount: AmountQueryParam, // amount which is filled in Transfer panel
     l2ChainId: NumberParam, // L2 chain-id with which we can initiaze (override) our networks/signer
-    token: StringParam // import a new token using a Dialog Box
+    token: StringParam, // import a new token using a Dialog Box
+    skipLoadingHistory: BooleanParam // internal - to not show pending transactions that spike our network calls during e2e tests
```

Comment:
A thought: should we maybe prefix it with something, e.g. `_skipLoadingHistory` or `INTERNAL_skipLoadingHistory` just so we don't accidentally use it outside of the test environment?

Reply:
I'm leaning towards `INTERNAL_skipLoadingHistory` as it's more explicit

Reply:
Makes sense - done.